k8s/overlays/harbor-registry/harbor-gateway-image.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apps/v1
```

Comment:
I don't think we need to use harbor registry in this public repo. Currently the animal rescue images are in our public docker hub team repo, so everyone has access to try this out.